database/migrations/2018_06_22_111322_create_posts_db.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreatePostsDb extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts_data', function (Blueprint $table) {
+            $table->increments('id')->unique();
```

Comment:
The increment type column sets it's values to be unique so we don't need to define it here. Please remove it.